### ISSUE TITLE
Beta stage errors

### ISSUE THREAD

**Issue by jsjtotw** [UnnormalPerson/IAmBored-_-#1]
### user() Function

A function called `user()` is defined. It initializes your team's health (`user_team`) to 5 and sets the attack strength of your team (`team_atk`) to 4. Then, it checks the value of `user_team` using conditional statements and adjusts the attack strength accordingly. If your team's health is greater than 84, the attack strength becomes 8. If it's greater than 64 but not greater than 84, the attack strength becomes 7, and so on. However, there's an issue with this function: it doesn't return anything, so the changes in `team_atk` inside this function won't affect the rest of the code.

### offense() Function

Next, there's a function `offense()` defined, but there's an error in its implementation. Instead of `random.radint`, it should be `random.randint` to generate a random number between 1 and 6 multiplied by the `team_atk` value. However, similar to the `user()` function, `offense()` doesn't return anything, so the calculated attack strength (`user_atk`) will not be accessible outside this function.

### Multi-line String (offense_who)

A multi-line string (`offense_who`) is defined for the attack loop, but it contains several issues. Instead of `random.radint`, it should be `random.randint`. Also, the variable `atk_who` should be obtained using the `input()` function to get the user's choice. The `elif ValueError` is incorrect and won't work as intended. To handle invalid input, it should be an `except ValueError:` block instead.

In conclusion, the code contains multiple issues and won't run as expected due to incorrect usage of functions, variable scopes, and loops. It requires significant reworking and fixing to make it a functioning text-based game.

**Comment by randomusernameplswork** [comment]
You screwed the last part

**Comment by jsjtotw** [comment]
> 

SHut up thiam thiam ni de ma ma mei you mao

**Comment by randomusernameplswork** [comment]
Do not reveal our real names, remember.

**Comment by randomusernameplswork** [comment]
Also thanks I'll work on it thanks for pulling an issue

**Comment by jsjtotw** [comment]
no one cares gay thiam thiam

**Comment by randomusernameplswork** [comment]
@jsjtotw please check the code now ty

**Comment by jsjtotw** [comment]
ok gay female thiam thiam

**Comment by randomusernameplswork** [comment]
I deleted it
